### PR TITLE
Change password

### PR DESCRIPTION
**Pull request by irina-baeva**
handle password change
arrange user settings screen
handle delete account
fix sign in sign up

### CODE REVIEW

File: src/screens/ChangePassword/index.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+  Text,
+  TextInput,
+  StyleSheet,
+  TouchableOpacity,
+  Button,
+} from 'react-native';
+import Modal from 'react-native-modal';
+import { Context as AuthContext } from '../../state/AuthContext';
+
+export default function SignUpScreen({ navigation }) {
```
Comment (thdvbr) [inline src/screens/ChangePassword/index.js:13]: why is this component called SignUpScreen? shouldn't it be ChangePassword?

File: src/screens/DeleteAccount/index.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
+import React, { useContext } from 'react';
+import { View, Text, StyleSheet, TouchableOpacity } from 'react-native';
+import { Context as AuthContext } from '../../state/AuthContext';
+
+export default function SignUpScreen({}) {
```
Comment (thdvbr) [inline src/screens/DeleteAccount/index.js:5]: This one also

File: src/screens/ChangePassword/index.js (hunk 1 above)
Comment (thdvbr) [reply src/screens/ChangePassword/index.js:13]: or is it because it's inside Signupscreen ?

Comment (irina-baeva) [reply src/screens/ChangePassword/index.js:13]: true, i will fix them both

**Review by thdvbr — APPROVED** [review]

**Pull request merged**